docs/cookbook/attachments.rst

```diff
@@ -76,12 +76,17 @@ descriptions of each field.
 
 - **image** (:obj:`str`):
     The url location of the thumbnail image assigned to this Attachment. For uploads, Shotgun
-    automatically tries to create a thumbnail from the file. Alternatively, you can assign your
+    automatically tries to create a thumbnail from the file.
+    See :ref:`interpreting_image_field_strings`. Alternatively, you can assign your
     own thumbnail to an Attachment using the :meth:`~shotgun_api3.Shotgun.upload_thumbnail` method.
 
 - **sg_type** (:obj:`str`):
     An optional field for designating different types of Attachments
 
+- **processing_status** (:obj:`str`):
+    Reflects the status of the attachment (File entity).
+    When processing the thumbnail, this field is set to ‘Thumbnail Pending’.
```

Comment:
Should we explain how to use this field? I thought we were going to have a helper method that you could pass an image to and it would tell you if the image was ready or not?

Reply:
Sorry, Github says I'm requesting for changes. I don't know why that is. I'm pretty sure I clicked "Comment" instead of Request Change.